manifests/a/app/vleer/1.0.0-beta.3/app.vleer.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3

```

```suggestion
PackageVersion: 1.0.0
```
Please update file name and path to match this change.